tests/test_logsumexp.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import pytest
+from scipy.special import logsumexp
+
+from timemachine.lib import custom_ops
+
+
+@pytest.mark.memcheck
+@pytest.mark.parametrize("precision,atol,rtol", [(np.float64, 1e-8, 1e-8), (np.float32, 2e-5, 1e-5)])
+@pytest.mark.parametrize("seed", list(range(2)))
+@pytest.mark.parametrize("size", [2, 100, 10000])
+def test_cuda_logsumexp(precision, atol, rtol, seed, size):
+    rng = np.random.default_rng(seed)
+
+    values = rng.normal(size=size) * 1000.0
```

Comment:
test should be modified to include both a set of very large values and a set of very negative values, i.e.:

`values = rng.normal(size=size, loc=-10000, scale=1.0)`
and
`values = rng.normal(size=size, loc=10000, scale=1.0)`

Reply:
Added in https://github.com/proteneer/timemachine/pull/1176/commits/cd982aae818703cf3cea0ac33c0e4e89d6bd6e1a